docs/bucket/notifications/README.md

```diff
@@ -222,12 +220,12 @@ import paho.mqtt.client as mqtt
 
 def on_connect(client, userdata, flags, rc):
   print("Connected with result code "+str(rc))
-  client.subscribe("minio")
+  client.subscribe("minio", 1)
 
 def on_message(client, userdata, msg):
     print(msg.payload)
 
-client = mqtt.Client()
+client = mqtt.Client(client_id="myclientid",clean_session=False)
```

Comment:
how would they know clientID when its autogenerated?

Reply:
This is the subscriber's client ID. It can be any random ID. We dont need the publisher's id here.

Reply:
yeah but can we add a comment on what this id is ?

Reply:
ok. Done :+1: